### DISCUSSION TITLE
Tag latest not pulling latest version?

### DISCUSSION THREAD

**Discussion by mbc0** [doitandbedone/ispyagentdvr-docker#552] [General]
Hi, 

I am using doitandbedone/ispyagentdvr:latest but it is pulling version 4.4.6.0? it is not pulling even 4.4.8.0 let alone 4.4.9.0 is this normal?

Thank you

**Comment by doitandbedone** [comment]
Images take time to be approved and to be processed in dockerhub. Check available versions here: https://hub.docker.com/repository/docker/doitandbedone/ispyagentdvr

**Comment by totalitarian** [comment]
Is the latest 4.7.2.0? Because that is the version that keeps getting pulled for me...
